src/test/java/com/google/api/codegen/testdata/discoveries/java/java_translate.v2.json.baseline

```diff
@@ -16,37 +16,27 @@ import com.google.api.client.http.HttpTransport;
 import com.google.api.client.json.JsonFactory;
 import com.google.api.client.json.jackson2.JacksonFactory;
 import com.google.api.services.translate.Translate;
-import com.google.api.services.translate.Translate.Detections.List;
 import com.google.api.services.translate.model.DetectionsListResponse;
 import java.io.IOException;
 import java.security.GeneralSecurityException;
 import java.util.ArrayList;
+import java.util.List;
 
 public class TranslateExample {
-  public static void main(String[] args) throws IOException, GeneralSecurityException {
-    Credential credential = getAuth();
-
-    HttpTransport httpTransport = GoogleNetHttpTransport.newTrustedTransport();
-    JsonFactory jsonFactory = JacksonFactory.getDefaultInstance();
-
-    Translate translateService = new Translate.Builder(httpTransport, jsonFactory, credential)
-        .setApplicationName("Google Cloud Platform Sample")
-        .build();
-
-
-    // TODO: Change placeholder below to desired parameter value for the `list` method:
-
+  public static void main(String args[]) throws IOException, GeneralSecurityException {
     // The text to detect
-    java.util.List<String> q = new ArrayList<>(); // TODO: Update placeholder value.
+    List<String> q = new ArrayList<>();  // TODO: Update placeholder value.
 
-    Translate translateService = createService();
-    List request = translateService.detections().list(q);
+    Translate translateService = createTranslateService();
+    Translate.Detections.List request = translateService.detections().list(q);
 
     DetectionsListResponse response = request.execute();
+
+    // TODO: Change code below to process the `response` object:
     System.out.println(response);
   }
 
-  public Translate createService() throws IOException, GeneralSecurityException {
+  public static Translate createTranslateService() throws IOException, GeneralSecurityException {
     HttpTransport httpTransport = GoogleNetHttpTransport.newTrustedTransport();
     JsonFactory jsonFactory = JacksonFactory.getDefaultInstance();
 
```

Comment:
This comment seems to have disappeared in relation to the baseline... so I assume you're talking about the same issue that #570 is resolving, so yes.